fastlane/fastlane
# [action] import_from_git: Import multiple files from git

### Checklist

### Motivation and Context
Currently, if you need to import multiple files from git - you will clone the repository for every single file.

### Description
I've checked that previous solution still works in a same way as before and that you will have access to lanes/defs that are located in fastfiles that we added as `dependencies` and not as `path` variable.

### Testing Steps
1. Add few Fastfiles to external git repo. 
2. Import them in previous way (example):
```
import_from_git(
  url: github_infra_repo_url,
  branch: github_infra_repo_branch,
  version: github_infra_repo_version,
  path: "fastlane/defs/Fastfile",
)

import_from_git(
  url: github_infra_repo_url,
  branch: github_infra_repo_branch,
  version: github_infra_repo_version,
  path: "fastlane/git-commands/Fastfile",
)

import_from_git(
  url: github_infra_repo_url,
  branch: github_infra_repo_branch,
  version: github_infra_repo_version,
  path: "fastlane/github-commands/Fastfile",
)

import_from_git(
  url: github_infra_repo_url,
  branch: github_infra_repo_branch,
  version: github_infra_repo_version,
  path: "fastlane/bump-versions/Fastfile",
)

import_from_git(
  url: github_infra_repo_url,
  branch: github_infra_repo_branch,
  version: github_infra_repo_version,
  path: "fastlane/Fastfile",
)
```
3. Check if you are able to use lanes/defs from those files
4. Change it to use import_from_git only once and add all additional files to `dependencies` and check if lanes still work
```
import_from_git(
  url: github_infra_repo_url,
  branch: github_infra_repo_branch,
  version: github_infra_repo_version,
  path: "fastlane/Fastfile",
  dependencies: ["fastlane/defs/Fastfile", "fastlane/git-commands/Fastfile", "fastlane/github-commands/Fastfile", "fastlane/bump-versions/Fastfile"]
)

lane :test do
  run_action_from_file_1
  run_action_from_file_2
  run_action_from_fastlane_Fastfile
end
```
